analysis/db/max/README.md

```diff
@@ -0,0 +1,8 @@
+I think that the following questions might also be analyzed using available dataset, starting with titles and abstracts of the articles.
+
+-What sources, best practices or challenges are in medical care to prevent spreading
+the SARS-CoV-2? (There was task in [Kaggle] as well)
+
+-What are the changes in people's behavior during this COVID-19 pandemic period?
```

Comment:
Where do you plan on getting (text?) data to look at "behavior"?

Reply:
I think it is possible to get from articles, like maybe what products are causing higher demand?
In my opinion, we can also see how offices/universities/other organizations react to this situation from their announcements. Besides, what platforms are causing more demand like zoom (I have recently read an article how Zoom has the highest stock market price), which means that more people started working/studying from home. Also, I am not sure if it's possible to get some data from posts in social media.